app/views/applications/index.html.slim

```diff
@@ -1,3 +1,3 @@
 / include webpacked react-table CSS
 == stylesheet_pack_tag 'react-table'
-== react_component 'ApplicationsPage', { applications: @applications, fields: @fields }
+== react_component 'ApplicationsPage', { listings: @listings }
```

Comment:
We do not pass applications from Rails anymore because we load it with ajax, since we have the eager pagination and the filter. I pass the listings, so I can populate the select in the filter with listings